pkg/kubectl/cmd/cp/cp.go

```diff
@@ -137,6 +140,14 @@ func extractFileSpec(arg string) (fileSpec, error) {
 				File:         file,
 			}, nil
 		}
+		if len(pieces) == 3 {
```

Comment:
use a constant for 3 with meaningful name or add a comment explaining why 3?

Reply:
I think we can just use "3" here, as the code before me is using "1" and "2". The code is quite straightforward and this style is throughout the code base of kubernetes.